### PR TITLE
Add GetDefaultMTU function

### PR DESCRIPTION
**Pull request by rcarrillocruz**
That function is currently in CNO tree, however we want to use it
on origin too, thus libifying it on library-go to avoid duplications
of code.

### CODE REVIEW

**Comment by danwinship** [comment]
You need to pass `--strip-vendor` to `glide`. Or rather, just use `make update-deps`.

File: glide.yaml [hunk 1]
```diff
@@ -57,3 +57,6 @@ import:
 # matches openshift/origin
 - package: github.com/gonum/graph
   version: 50b27dea7ebbfb052dfaf91681afc6fde28d8796
+
+- package: github.com/vishvananda/netlink
```
Comment (smarterclayton) [inline glide.yaml]: This dependency really doesn’t belong in library-go.  Pulling this dependency into library go to reuse a 20 line function is overkill

**Comment by smarterclayton** [comment]
/hold

I don’t think this belongs in library go.  A 20 line function is fine to copy

**Comment by danwinship** [comment]
ok

**Pull request closed without merging**